### PR TITLE
Adjusted a few naming conventions and normalized some styles

### PR DESCRIPTION
**Pull request by eskirk**
Just touched up some code to match some of @sccolbert 's recommendations.

### CODE REVIEW

**Comment by blink1073** [comment]
Is this meant to replace #127?

**Comment by sccolbert** [comment]
looks to be a replacement

File: src/notebook/index.css [hunk 1]
```diff
@@ -37,67 +37,87 @@
...
+.jp-MarkdownCell-content.p-Widget h4 {
   font-family: "Helvetica Neue", Arial, Helvetica, Geneva, sans-serif;
   font-size: 14px;
   line-height: 1.0;
+  margin: 14px 0 0 0;
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget h5 {
+.jp-MarkdownCell-content.p-Widget h4 {
```
Comment (sccolbert) [inline src/notebook/index.css]: should be h5?

Comment (sccolbert) [inline src/notebook/index.css]: actually this looks like a duplicate rule

Comment (eskirk) [inline src/notebook/index.css]: Yeah that was an error, I changed it. 
My b, sorry about that.

**Comment by eskirk** [comment]
Yeah it is supposed to be a replacement. Is there a way to specify that?

**Comment by sccolbert** [comment]
No way to specify. Just close the old issue with a comment, like I did.

File: src/notebook/index.css [hunk 2]
```diff
@@ -37,67 +37,87 @@
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget {
+.jp-MarkdownCell-content.p-Widget {
+  padding-left: 5.6px;
```
Comment (sccolbert) [inline src/notebook/index.css]: A fractional pixel value doesn't make much sense, except for hi-dpi displays, and then only in extreme cirumstances. Any particular reason this is fractional?

File: src/notebook/index.css [hunk 3]
```diff
@@ -37,67 +37,80 @@
...
+.jp-MarkdownCell-content.p-Widget h4 {
   font-family: "Helvetica Neue", Arial, Helvetica, Geneva, sans-serif;
   font-size: 14px;
   line-height: 1.0;
+  margin: 14px 0 0 0;
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget h5 {
+
```
Comment (sccolbert) [inline src/notebook/index.css]: extraneous blank line

**Comment by sccolbert** [comment]
LGTM modulo my two most recent comments

File: src/notebook/index.css (hunk 2 above)
Comment (eskirk) [inline src/notebook/index.css]: I was just matching what I saw in the notebook, no particular reason. I can change it if you'd like.

Comment (sccolbert) [inline src/notebook/index.css]: I'd probably just go with 7px all around for simplicity: `padding: 7px`

Comment (sccolbert) [inline src/notebook/index.css]: Though I do remember @ellisonbg talking about making everything a multiple of 4 at some point.

Comment (eskirk) [inline src/notebook/index.css]: I'll change it to 7 for now and run it by @ellisonbg when he comes into the office later

File: src/notebook/index.css [hunk 4]
```diff
@@ -37,67 +37,79 @@
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget {
+.jp-MarkdownCell-content.p-Widget {
+  padding-left: 7px;
+  padding-top: 7px;
+  padding-right: 7px;
+  padding-bottom: 7px;
```
Comment (sccolbert) [inline src/notebook/index.css]: One last niggle. The shorthand `padding: 7px` can be used to set everything to the same value. Thanks for being patient!

**Comment by sccolbert** [comment]
:shipit:

**Pull request merged**